template/package.json

```diff
@@ -141,7 +145,7 @@
     "build:sass": "npm-run-all build:sass:demo build:sass:component postCss:component sass:render",
     "build:sass:demo": "sass demo:demo",
     "build:sass:component": "sass src:src",
-    "build:watch": "nodemon -e scss,js --watch src --exec npm run build:dev:assets",
+    "build:watch": "nodemon -e scss,js,md,html --watch src --exec npm-run-all prettier:write build:dev:assets",
```

Comment:
Set up this way, we are enforcing the opinion that running `npm run dev` you will be running the prettier writer on save. Do we want that? 

Let's also keep in mind that not all devs run `dev` and `serve`. Up till this point, unless you are updating Sass, you don't really need the `dev` script running. I feel `prettier:check` should be in the Huskey pre-commit so that any errors are caught. I am not sure how, but it would be nice if we could get a message in there that if there are issues, they can run `npm run prettier:write"` to address issues. 

Per @geoffrich feedback about how this butts up against the linters, I am wondering if we run this with the `linters` command as well? Run the `prettier:check` command again with the instruction to run `npm run prettier:write"` to fix.